tests/behaviour/steps/inspectcli.py

```diff
@@ -171,3 +219,341 @@ def step_impl(context, error_message: str):
     except Exception as ex:
         assert ex is not None
         assert ex.message == f"{error_message} {context.csvw_metadata_file}"
+
+
+@Then("the Type printable is validated for single-measure pivoted data set")
+def step_impl(context):
+    result_type_info: CSVWType = context.result_type_info
+    assert result_type_info is not None
+    assert result_type_info == CSVWType.QbDataSet
+
+
+@Then("the Catalog Metadata printable is validated for single-measure pivoted data set")
+def step_impl(context):
+    result_catalog_metadata: CatalogMetadataResult = context.result_catalog_metadata
+    assert result_catalog_metadata is not None
+    assert result_catalog_metadata.title == "Pivoted Shape Cube"
+    assert result_catalog_metadata.label == "Pivoted Shape Cube"
+    assert result_catalog_metadata.issued == "2022-10-28T13:35:50.699296"
+    assert result_catalog_metadata.modified == "2022-10-28T13:35:50.699296"
+    assert result_catalog_metadata.license == "None"
+    assert result_catalog_metadata.creator == "None"
+    assert result_catalog_metadata.publisher == "None"
+    assert (
+        len(result_catalog_metadata.landing_pages) == 1
+        and result_catalog_metadata.landing_pages[0] == ""
+    )
+    assert (
+        len(result_catalog_metadata.themes) == 1
+        and result_catalog_metadata.themes[0] == ""
+    )
+    assert (
+        len(result_catalog_metadata.keywords) == 1
+        and result_catalog_metadata.keywords[0] == ""
+    )
+    assert result_catalog_metadata.contact_point == "None"
+    assert result_catalog_metadata.identifier == "qb-id-10004"
```

Comment:
Should this be an input to the step definition? 

Can you either reword the step definition so it takes all the values which might change as parameters, or make the step name so specific no one else will attempt to reuse it?